src/main/java/strategy/ISale.java

```diff
@@ -0,0 +1,5 @@
+package strategy;
+
+public interface ISale {
+    Boolean sell(double amount);
```

Comment:
instead of bool I'd be more interesting to have the amount of money it's being charged for the sale.